src/features/surveys/components/surveyForm/SurveyForm.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+
+import { Box } from '@mui/material';
+import { FC } from 'react';
+import { submit } from 'features/surveys/actions/submit';
+import SurveyElements from './SurveyElements';
+import SurveyHeading from './SurveyHeading';
+import SurveyPrivacyPolicy from './SurveyPrivacyPolicy';
+import SurveySignature from './SurveySignature';
+import SurveySubmitButton from './SurveySubmitButton';
+import SurveySuccess from './SurveySuccess';
+import {
+  ZetkinSurveyExtended,
+  ZetkinSurveyFormStatus,
+} from 'utils/types/zetkin';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
What's the problem with this import that requires `ts-ignore`?

Reply:
Added an explanatory comment for future readers. Ideally the React team would publish typedefs directly within these libs. The DefinitelyTyped definitions for react-dom are lagging behind these new canary version features.